aea/protocols/generator.py

```diff
@@ -51,6 +51,26 @@
     "bool": "bool",
     "str": "string",
 }
+FETCHAI_COPYRIGHT = (
```

Comment:
I don't think we should add fetchai as the copy holder. Instead, it should be the `author`